pages/about.html

```diff
@@ -4,9 +4,43 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>✨About Me✨</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  
 </head>
+<header>
+  <h1>K. Buslach </h1>
+  <h1>ADA C17 Sharks 🦈</h1>
```

Comment:
It depends on who you ask, but your document should have only one h1 element. Of course you can add multiple and the browser will still render the h1 tags, but it is more semantically correct to only use 1 since heading tags have a hierarchy